### ISSUE TITLE
0058-feel-number-function

### ISSUE THREAD

**Issue by saig0** [camunda/tck#7]
```
Failed to parse FEEL expression 'number(from: "1.000.000,01", decimal sep:",", grouping sep:".")': failed to parse expression 'number(from: "1.000.000,01", decimal sep:",", grouping sep:".")': Expected (StringIn("<=", ">=", "<", ">", "!=", "=") | "between" | "in" | "and" | "or" | "," ~ namedParameter | ")"):1:28, found ", decimal "
```

Test: invalid function argument names.

**Comment by saig0** [comment]
https://github.com/camunda/feel-scala/issues/311

**Comment by saig0** [comment]
The issue is fixed with DMN-Scala version 1.7.4.
